artifact.md

```diff
@@ -15,11 +15,15 @@ For the media type(s) that this is compatible with see the [matrix](media-types.
 
   This property MUST be used and contain the media type `application/vnd.oci.artifact.manifest.v1+json`.
 
-- **`blobs`** *string*
+- **`artifactType`** *string*
 
-  This OPTIONAL property contains a list of [descriptors](descriptor.md). Each descriptor represents an artifact of any IANA mediaType. The list MAY be ordered for certain artifact types like scan results.
+  This REQUIRED property specifies the mediaType of the referenced content and MAY be registered with [IANA][iana].
```

Comment:
Whether this is  OPTIONAL or REQUIRED needs to be discussed on the call.

Reply:
I believe it was previously determined to be required based on preventing recent mediatype security issue

Reply:
I lean towards `SHOULD` instead of `REQUIRED`